chalice/app.py

```diff
@@ -266,15 +292,42 @@ def __init__(self, body, headers=None, status_code=200):
         self.headers = headers
         self.status_code = status_code
 
-    def to_dict(self):
+    def to_dict(self, binary_types=None):
```

Comment:
No, this is not typically used by an end user (but could be if they wanted), it's used by the chalice handler.